cloud/resource_cloud_environment.go

```diff
@@ -106,6 +106,16 @@ func resourceCloudEnvironment() *schema.Resource {
 					},
 				},
 			},
+			"wait_for_completion": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: descriptions["wait_for_completion"],
+			},
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
```

Comment:
@maxsxu I've set the default TF timeout to 60 minutes, do we think this is a good value?

Reply:
LGTM